common/djangoapps/student/forms.py

```diff
@@ -211,6 +211,15 @@ def __init__(
             if field not in self.fields:
                 self.fields[field] = forms.CharField(required=False)
 
+    def clean_email(self):
+        """
+        Checks is there already registered user with the given email
+        """
+        email = self.cleaned_data["email"]
+        if User.objects.filter(email__iexact=email).exists():
+            raise ValidationError(_("An account with the Email '{email}' already exists.").format(email=email))
```

Comment:
@douglashall, some python tests are failing due to this change in error message. Should I discard this change or change the unittests ?